docs/reference/commandline/cli.md

```diff
@@ -96,28 +96,36 @@ variables.
 ### Configuration files
 
 By default, the Docker command line stores its configuration files in a
-directory called `.docker` within your `$HOME` directory. However, you can
-specify a different location via the `DOCKER_CONFIG` environment variable
-or the `--config` command line option. If both are specified, then the
-`--config` option overrides the `DOCKER_CONFIG` environment variable.
-For example:
-
-    docker --config ~/testconfigs/ ps
-
-Instructs Docker to use the configuration files in your `~/testconfigs/`
-directory when running the `ps` command.
+directory called `.docker` within your `$HOME` directory.
 
 Docker manages most of the files in the configuration directory
 and you should not modify them. However, you *can modify* the
 `config.json` file to control certain aspects of how the `docker`
 command behaves.
 
-Currently, you can modify the `docker` command behavior using environment
+You can modify the `docker` command behavior using environment
 variables or command-line options. You can also use options within
 `config.json` to modify some of the same behavior. When using these
-mechanisms, you must keep in mind the order of precedence among them. Command
-line options override environment variables and environment variables override
-properties you specify in a `config.json` file.
+mechanisms, you **must** keep in mind the order of precedence among them.
+Command line options override environment variables and environment variables
+override properties you specify in a `config.json` file.
+
+
+#### Change the `.docker` directory
+
+To specify a different directory, use the `DOCKER_CONFIG`
+environment variable or the `--config` command line option. If both are
+specified, then the `--config` option overrides the `DOCKER_CONFIG` environment
+variable.
+
+For example:
+
+    docker --config ~/testconfigs/ ps
```

Comment:
Can you add `bash` code-hints and a prompt (`$`) as we do in other shell examples?

```suggestion
```bash
$ docker --config ~/testconfigs/ ps
```

Perhaps we should use a (possibly common) location for the example (if people like to use an `XDG` directory structure (https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html);

```suggestion
```bash
$ docker --config ~/.config/docker/ ps
```

(or `docker --config ${XDG_CONFIG_HOME}/docker ps`, but that might be too specific to XDG)

Reply:
I think think that's too specific for this example.